File: pset6/mario/less/mario.py
from cs50 import get_int
from sys import exit

while True: 
    x = get_int('Height: ')     # Get Input
    if x > 0 and x < 9:     # Condition
        cnt = x - 1     # Spaces
        for i in range(x):
            for j in range(cnt):
                print(' ', end='')      # Print Spaces
                # print('\n')
            for z in range(cnt, x):
                print("#", end='')       # Print Symbols--> '#'
            print('')       # New Line
            cnt -= 1
        exit(0)     # return 0

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i = 0, j = 0, cnt = 0, z = 0, x = 0;
    // printf("Height: ");
    // scanf("%d", &x);
    do
    {
        x = get_int("Height: ");    // Gets Input
    }
    while (x > 8 || x < 1); // Checking Iput valuee
    j = x;
    cnt = x - 1;    // spaces
    for (i = 0; i < x; ++i) // loop for rows
    {
        for (j = 0; j < cnt; ++j) // loop for columns
        {
            printf(" ");    // Printing Spaces
        }
        for (z = cnt; z < x; ++z)   // loop for columns
        {
            printf("#");     // Printing Hash Symbols
        }
        printf("\n");   // New Line
        --cnt;  // Spaces Count in each column
    }               
}
